lib/mwUtil.js

```diff
@@ -300,7 +300,10 @@ mwUtil.decodeBody = (contentResponse) => {
  */
 mwUtil.getSiteInfo = (hyper, req, domain) => hyper.get({
     uri: new URI([domain || req.params.domain, 'sys', 'action', 'siteinfo'])
-}).get('body');
+}).get('body').then((siteInfo) => {
```

Comment:
Actually, let's go one step further and `toString` it in action.js?

Reply:
Done.